lib/site_prism/element_container.rb

```diff
@@ -69,6 +69,18 @@ def raise_if_block(obj, name, has_block)
 
     private
 
+    def combine_args(arg_array1, arg_array2)
+      args = []
+      args += arg_array1
+      options1 = (args.pop if args[-1].is_a?(Hash)) || {}
```

Comment:
> It is probably worth a refactor at some point.

I wasn't sure where to put this function really.  This seemed semi-reasonable at the time.